Integrations/src/main/java/io/deephaven/integrations/learn/FutureOffset.java

```diff
@@ -35,9 +32,10 @@ public FutureOffset(Future future, int offset) {
      */
     public int getOffset() { return offset; }
 
-    /** Gets the result of the deferred calculation.
+    /**
+     * Gets the result of the deferred calculation.
      *
      * @return the result of deferred calculation.
      */
-    public PyObject getFutureGet() { return future.get(); }
+    public Object getDeferredCalculation() { return future.get(); }
```

Comment:
I'm looking over the code and trying to figure out how to make it less confusing for the next person.  Given the recent changes, I think this can best be done by:
1) Make `Future` public
2) Make `Future.get()` public
3) Remove this method and replace it with `public Future getFuture()`

I understand that this seems to be some backpedalling on a few things, but I think it is the best option given your changes.